<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigGroupState</class>
 <widget class="QWidget" name="ConfigGroupState">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background:#35363C;
font-size:14px;
color:white;
}
QWidget:disabled{
color:#7F7F7F;
}
QLineEdit{
border:none;
border-bottom:1px solid #5A636C;
padding-left:8px;
padding-right:8px;
}
QLineEdit:hover{
background:#4C4E56;
}
QLineEdit:focus{
border:1px solid #02BFFF;
}
QLabel:enabled[mstyle=&quot;hint&quot;]{
font-size:12px;color:#A0C2D3;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="line_config_box" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="1">
       <widget class="QLineEdit" name="input_signal_check">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_trigger">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>触发信号：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="input_trigger_func">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_signal_check">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>信号参数检查：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_signal_check_hint1">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>*设置信号参数检查函数，返回true时表示信号有效</string>
        </property>
        <property name="mstyle" stdset="0">
         <string>hint</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_trigger_hint">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>*触发该分支的信号函数</string>
        </property>
        <property name="mstyle" stdset="0">
         <string>hint</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_signal_check_hint2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>*函数签名：
Q_INVOKABLE bool function(int branchId,const QVariantList&amp; data)</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="mstyle" stdset="0">
         <string>hint</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>标记为：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="check_next_branch">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>下一个分支</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">branch_type_group</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_sub_state_branch">
          <property name="text">
           <string>子分支</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">branch_type_group</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="check_fail_branch">
          <property name="text">
           <string>失败分支</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">branch_type_group</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>384</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="branch_type_group"/>
 </buttongroups>
</ui>
